codewof/programming/content/structure/questions.yaml

```diff
@@ -485,3 +485,161 @@ drawn-out-string:
     2: normal
     3: normal
     4: normal
+
+leap-year:
+  types:
+    -program
+  test-cases:
+    1: normal
+    2: normal
+    3: normal
+    4: normal
+
+to-celsius:
+  types:
+    - function
+    - parsons
+  parsons-extra-lines:
+    - 'celsius = (fahrenheit + 32) * (5/9)'
+    - 'print(celsius)'
+    - 'round(fahrenheit)'
+  test-cases:
+    1: normal
+    2: normal
+    3: normal
+    4: normal
+
+driver-speed:
+  types:
+    - function
+    - parsons
+  parsons-extra-lines:
+    - 'if points > 5'
+    - 'if speed < 50'
+    - 'int(speed / 5)'
+  test-cases:
+    1: normal
+    2: normal
+    3: normal
+    4: normal
+
+fizz-buzz:
+  types:
+    - function
+    - parsons
+  parsons-extra-lines:
+    - 'divisible_by_3 = num / 3 == 0'
+    - 'divisible_by_5 = num / 5 == 0'
+    - "print('fizzbuzz')"
+    - 'if divisible_by_3 or divisible_by_5:'
+  test-cases:
+    1: normal
+    2: normal
+    3: normal
+    4: normal
+
+sum-multiples-3-5:
+  types:
+    - program
+  test-cases:
+    1: normal
+    2: normal
+    3: normal
+    4: normal
+
+prime-numbers:
+  types:
+    - program
+  test-cases:
+    1: normal
+    2: normal
+    3: normal
+    4: normal
+    5: normal
+
+merge-lists-odd-even:
+  types:
+    - function
+    - parsons
+  parsons-extra-lines:
+    - 'if (num / 2 == 0)'
+    - 'if (num / 2 != 0)'
+    - 'print(result)'
+    - 'result[-1] = num'
+  test-cases:
+    1: normal
+    2: normal
+    3: normal
+
+inside-25-to-50:
+  type: debugging
+  number_of_read_only_lines_top: 2
+  number_of_read_only_lines_bottom: 0
+  test-cases:
+    1: normal
+    2: normal
+    3: normal
+    4: normal
+
+merge-lists-odd-even:
```

Comment:
:O you've only got 12 questions, there's a duplicate
![image](https://user-images.githubusercontent.com/25916475/64929550-ac9c1d00-d87b-11e9-92be-4e608c72f1b4.png)